cmd/update-rules/main_test.go

```diff
@@ -163,3 +163,40 @@ func TestUpdateRules(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteRules(t *testing.T) {
+	tests := []struct {
+		name      string
+		branch    string
+		goVersion string
+	}{
+		{
+			"release-1.20",
+			"release-1.20",
+			"1.17.1",
+		},
+		{
+			"release-1.19",
+			"release-1.19",
```

Comment:
```suggestion
			"deleting rule for non existing branch",
			"release-1.25",
```

Reply:
Will be better if we can name the tests like this instead of just giving the branch name.